src/hooks/api/collections/useUpdateCollectionInfo.ts

```diff
@@ -46,6 +47,19 @@ export default function useUpdateCollectionInfo() {
         },
         false
       );
+
+
+      // Update single collection cache
+      await mutate(
+        getCollectionByIdCacheKey({ id: collectionId }),
+        (value: GetCollectionResponse) => {
+          const newValue = cloneDeep<any>(value);
+          newValue.collection.name = name;
+          newValue.collection.collectors_note = collectors_note;
+          return newValue
+        },
+        false
+      );
```

Comment:
thank you for dealing with our weird meme lol

Reply:
It works 😂